dive_core/generatePm4Info_adreno.py

```diff
@@ -823,6 +825,11 @@ def outputPacketInfo(pm4_info_file, registers_et_root, enum_index_dict, opcode_d
       # Keep track of instance #. Only the 1st instance belongs in the vector. The rest are in the multimap.
       if opcode not in packet_type_instances:
         packet_type_instances[opcode] = 1
+        if (opcode == 0):
+          pm4_info_file.write("    // For descriptors, we purposefully try to include them as \"packets\" for easier parsing.\n")
```

Comment:
PEP 8 encourages using single quotes here to avoid backslashes (https://peps.python.org/pep-0008/#string-quotes).

```suggestion
          pm4_info_file.write('    // For descriptors, we purposefully try to include them as "packets" for easier parsing.\n')
```

Reply:
Thanks for the suggestion, fixed